src/app.js

```diff
@@ -0,0 +1,17 @@
+const csv = require('csvtojson');
+const fs = require('fs');
+
+import { handleError } from './error-handle.js';
+
+const csvFilePath = './src/csv/node_mentoring_t1_2_input_example.csv';
+const txtFilePath = 'csvToJson.txt';
```

Comment:
I would suggest creating a function or class which will be responsible for all manipulations.